ml_peg/analysis/physicality/diatomics/analyse_diatomics.py

```diff
@@ -0,0 +1,403 @@
+"""Analyse diatomics benchmark."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from ml_peg.analysis.utils.decorators import (
+    build_table,
+    periodic_curve_gallery,
+)
```

Comment:
```suggestion
from ml_peg.analysis.utils.decorators import build_table, periodic_curve_gallery
```
Unimportant but saves a few lines